docs/section11/Rule 11-17.md

```diff
@@ -0,0 +1,49 @@
+# Service_Water_Heating - Rule 11-17  
+**Schema Version:** 0.0.37  
+
+**Mandatory Rule:** TRUE  
+
+**Rule ID:** 11-17  
+
+**Rule Description:** "All buildings that will have service water heating loads must include those loads in the simulation"  
+
+**Rule Assertion:** PASS/FAIL/UNDETERMINED  
+
+**Appendix G Section Reference:** Table G3.1 #1, proposed column, (a)  
+
+**Evaluation Context:** P_RMD - each building segment  
+**Data Lookup:**   
+**Function Call:**  
+- **get_SWH_bats_and_SWH_use**  
+- **get_building_segment_SWH_bat**  
+- **get_SWH_uses_associated_with_each_building_segment**  
+- **ge_component_by_id**  
+
+## Applicability Checks:  
+
+- look at each building in the proposed RMD: `for building in P_RMD:`
+    - look at each building segment: `for building_segment in building:`
+        - we want to look at all building segments, so continue directly to the rule logic: `CONTINUE TO RULE LOGIC`
+      
+        ## Rule Logic:
+        - get the building segment service water heating building area type: `swh_bat = get_building_segment_SWH_bat(P_RMD,building_segment)`
+        - check if the building segment has service water heating loads.  First create a boolean has_swh_loads and set it to false: `has_swh_loads = false`
+        - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+        - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+            - get the swh_use using get_component_by_ID: `swh_use = get_component_by_ID(P_RMD, swh_use_id)`
+            - check to see if the use has SWH loads: `if swh_use.use > 0:`
+                - set has_swh_loads to TRUE and break out of the loop: `has_swh_loads = TRUE; break;`
+
+
+
+        ## Rule Assertion:
+        - if the building segment has SWH loads, PASS: `if has_swh_loads: PASS`
+        - if the BAT is UNDETERMINED, then rule outcome is UNDETERMINED and provide a note: `elif swh_bat == 'UNDETERMINED': UNDETERMINED; note = "The SWH Building Area type is UNDETERMINED, so this rule cannot be evaluated.` 
```

Comment:
Recommend makign the note more detailed. "No SWH loads were simulated. The SWH Building Area type is UNDETERMINED, so this rule cannot assess whether building type is likely to have SWH loads. Recommend manual check to determine if SWH loads should have been simulated based on whether the building will have SWH loads."